views/pages/toplevel/home.njk

```diff
@@ -25,75 +28,73 @@
 {% block content %}
 
     {% set heroContent %}
-        {{ headers.overlayText('h2', "The Big Lottery Fund is the largest funder of community activities in the UK. We distribute money raised by players of the <a href=\"https://www.national-lottery.co.uk\">National Lottery</a>", 'overlay-text--has-links t2') }}
+        <div class="o-button-group">
+            <a class="btn btn--small [ accent--white a--btn ]"
+                href="{{ buildUrl('toplevel', 'under10k') }}">
+                <span class="accent--pink a--text">
+                    <span class="u-wide-only">Learn how to apply for funding under £10,000</span>
+                    <span class="u-not-wide">Apply for funding under £10,000</span>
+                </span>
+            </a>
+
+            <a class="btn btn--small [ accent--white a--btn ]"
+                href="{{ buildUrl('toplevel', 'over10k') }}">
+                <span class="accent--blue a--text">
+                    <span class="u-wide-only">Learn how to apply for funding over £10,000</span>
```

Comment:
Not sure how great the double text is here. Personally I'd prefer to use the shorter button label everywhere. I've made a note to bring that question up.